usage/plugins/org.jboss.tools.usage/src/org/jboss/tools/usage/internal/environment/eclipse/IEclipseEnvironment.java

```diff
@@ -8,15 +8,15 @@
  * Contributors:
  *     Red Hat, Inc. - initial API and implementation
  ******************************************************************************/
-package org.jboss.tools.usage.googleanalytics.eclipse;
+package org.jboss.tools.usage.internal.environment.eclipse;
 
-import org.jboss.tools.usage.googleanalytics.IGoogleAnalyticsParameters;
+import org.jboss.tools.usage.internal.environment.IUsageEnvironment;
 
 /**
  * 
  * @author Andre Dietisheim
  *
  */
-public interface IEclipseEnvironment extends IGoogleAnalyticsParameters {
+public interface IEclipseEnvironment extends IUsageEnvironment {
 	public IEclipseUserAgent getEclipseUserAgent();
```

Comment:
why have an eclipse env that extends a general purpose env? Is this really necessary (are we ever going to run this in something that's not eclipse?)?